exec.js

```diff
@@ -69,9 +77,10 @@ function checkNotis() {
 			throw error
 		}
 		for (noti of data) {
-			if (noti.status
-					&& noti.status.content
-					&& noti.status.content.includes('go now')) {
+			if ((ownAcct !== undefined || noti.account.acct !== ownAcct)
```

Comment:
Shouldn't `ownAcct !== undefined` be `ownAcct == undefined`?

Currently it reads "If our own account has been found out, don't bother checking if they match." Which has the result that in most(all, essentially) cases, no check is made.

When I believe you mean to read "If our own account has *not* been found out, just go ahead and read the notification"

Reply:
Incidentally, the technical ""proper"" way to check undefined is

    typeof ownAcct == "undefined"

but I don't really care XP

Reply:
Alternatively, if we want to *ignore* notifications before its been found out, we can do
```suggestion
			if (noti.account.acct !== ownAcct
```

Which is a little cleaner. As long as `checkNotis` isn't called immediately it'll be fine, because either way its undesired behavior.

Reply:
> Incidentally, the technical ""proper"" way to check undefined is
> 
> ```
> typeof ownAcct == "undefined"
> ```

https://stackoverflow.com/a/4725697
it seems that that's only necessary if ownAcct may not have been declared already.

Reply:
Ok that's fine. Either way we need to resolve the backwards conditional here